tools/benchmarks/llm_eval_harness/meta_eval/README.md

```diff
@@ -38,14 +37,20 @@ To access our [3.1 evals Hugging Face collection](https://huggingface.co/collect
 - Log in to the Hugging Face website and click the 3.1 evals dataset pages and agree to the terms.
 - Follow the [Hugging Face authentication instructions](https://huggingface.co/docs/huggingface_hub/en/quick-start#authentication) to gain read access for your machine.
 
+The same process can be followed to access the [3.2 evals Hugging Face collection](https://huggingface.co/collections/meta-llama/llama-32-evals-66f44b3d2df1c7b136d821f0)
+
 It is recommended to read the dataset card to understand the meaning of each column and use the viewer feature in the Hugging Face dataset to view our dataset. It is important to have some basic understanding of our dataset format and content before proceeding.
 
 ### Task Selection
 
-Given the extensive number of tasks available (12 for pretrained models and 30 for instruct models), here we will focus on tasks that overlap with the popular Hugging Face [Open LLM Leaderboard v2](https://huggingface.co/spaces/open-llm-leaderboard/open_llm_leaderboard) as shown in the following:
+Given the extensive number of tasks available (12 for pretrained models and 30 for instruct models), a subset of tasks are chosen:
 
-- **Tasks for pretrained models**: BBH and MMLU-Pro
-- **Tasks for instruct models**: Math-Hard, IFeval, GPQA, and MMLU-Pro
+- **Tasks for 3.1 pretrained models**: BBH and MMLU-Pro
+  - Chosen as they overlap with the Hugging Face [Open LLM Leaderboard v2](https://huggingface.co/spaces/open-llm-leaderboard/open_llm_leaderboard)
+- **Tasks for 3.2 pretrained models**: MMLU
+  - MMLU is a common eval, and is the first one shown on on the [llama website](https://llama.com)
```

Comment:
There aren't any tasks for llama 3.2 pretrain that overlap with the hugging face leaderboard:

<img width="563" alt="image" src="https://github.com/user-attachments/assets/237725a6-2eff-4027-a45c-d3dd7e895b47">

<img width="521" alt="image" src="https://github.com/user-attachments/assets/ffc9acf9-b3f5-4cb6-a0af-2f297f58ea7d">

So I just chose MMLU because it's common and it's the first one that appears on llama.com:

<img width="964" alt="image" src="https://github.com/user-attachments/assets/5dddff4d-1c49-467c-94b8-ff034997363b">

But lmk if you have any other preferences here and I can accommodate.